{"name": "Получение заметки по некорректному ID", "status": "passed", "steps": [{"name": "Попытка получения заметки с некорректным ID=!@#$", "status": "passed", "start": 1759765494508, "stop": 1759765494510}], "parameters": [{"name": "invalid_id", "value": "'!@#$'"}], "start": 1759765494508, "stop": 1759765494510, "uuid": "225d5283-0614-4bc7-aa40-074ade7b11c4", "historyId": "43df5bdf2ec361a70b80c5cba592b932", "testCaseId": "efeb17e83461f5117735527552790e0e", "fullName": "tests.test_api.TestNoteRetrieval#test_get_note_by_invalid_id", "labels": [{"name": "feature", "value": "Управление заметками"}, {"name": "story", "value": "Получение заметок"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestNoteRetrieval"}, {"name": "host", "value": "E4435"}, {"name": "thread", "value": "1432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestNoteRetrieval"]}